doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md

```diff
@@ -88,22 +92,27 @@ To implement the switch side functionality the following changes shall be done:
 - Config DB schema should be extended to include the following information:
   - Support of the new `SmartSwitch` subtype in DEVICE_METADATA config DB table. The new subtype shall allow to identify when the SONiC is running on the Smart Switch.
   - Support of the new `MID_PLANE_BRIDGE` table with the following information
-    - Midplane bridge configuration with the bridge IP address and list of the DPU PCIe interfaces that should be added to the bridge for the Smart Switch case.
+    - Midplane bridge configuration with the list of the DPU PCIe interfaces that should be added to the bridge for the Smart Switch case.
```

Comment:
@Pterosaur can we update line 89 to indicate its not hwsku.json but platform.json. @oleksandrivantsiv FYI